spotify/views.py

```diff
@@ -57,6 +57,10 @@ def user_playlist_tracks(request, playlist_id, playlist_name):
 
     for track in user_playlist_tracks['items']:
         list_of_track_ids.append(track['track']['id'])
+    
+    # check for playlist with no tracks
+    if not list_of_track_ids:
+        return redirect('room', room_id=request.user.active_room_id)
 
```

Comment:
I propose that we aim to keep our code [properly structured](https://docs.google.com/presentation/d/1UXdPOewrZr47a6sIHbArN26_EdfwUzyrY9dSRoLcal4/edit?usp=sharing) - as per my really useful presentation :). This isn't a structured form and it fails the criteria to be considered a guard clause. So I recommend putting the rest of the code inside an `else:`.

Reply:
Yes. Will do as it makes perfect sense.